aten/src/ATen/Declarations.cwrap

```diff
@@ -983,42 +983,6 @@
         - arg: bool keepdim
           default: "false"
 ]]
-[[
-  name: _th_any
-  types:
-    - Byte
-  variants:
-    - function
-  backends:
-    - CPU
-    - CUDA
-  options:
-    - cname: logicalAnyAll
```

Comment:
can you get rid of logicalAnyAll and logicalAnd in TH/THC?

Reply:
Sure, let me try. 

BTW, I haven't fully updated this PR yet, as it would share the same a vectorized implementation with `all(*)`. I will update it after the `all(*)` PR is merged. So, let me first try removing `logicalAndAll` in #15540